torchrl/data/tensordict/tensordict.py

```diff
@@ -67,6 +66,7 @@ class _TensorDict(Mapping, metaclass=abc.ABCMeta):
 
     _safe = False
     _lazy = False
+    is_meta = False
```

Comment:
not a big deal but you want to name this as `_meta` to be consistent with the rest of the code?

Reply:
it comes from the fact that we have checks whether a tensor has [an attribute "is_meta"](https://pytorch.org/docs/stable/generated/torch.Tensor.is_meta.html) (i.e. is it on "meta" device)
TensorDict does not support meta device...